core/src/main/python/wlsdeploy/aliases/model_constants.py

```diff
@@ -150,6 +150,7 @@
 JMX_NOTIFICATION = 'JMXNotification'
 JNDI_NAME = 'JNDIName'
 JNDI_PROPERTY = 'JNDIProperty'
+JPA = 'JPA'
```

Comment:
General question - should JPA be included as part of updateDomain? JTA and JMX are excluded, not sure about this one. Opinions?